### ISSUE TITLE
stdout catching for yadage

### ISSUE THREAD

**Issue by tiborsimko** [reanahub/reana-demo-root6-roofit#17]
In #14 and #15, the standard output of the two steps is being captured in `gendata.log` and `fitdata.log` for CWL and Serial workflows.

Let us add the stdout catching for Yadage workflows as well.  @lukasheinrich What would be the best paradigm to use?  We could use the `tee` command as for the Serial workflows, but perhaps there exist a more native paradigm like for CWL?

**Comment by lukasheinrich** [comment]
Does the job controller have a API call to get the stdout after the job has finished?

**Comment by diegodelemos** [comment]
Yes, there is https://reana-job-controller.readthedocs.io/en/latest/restapi.html#get--jobs-job_id-logs, we will offer this through the rest API in this quarter.
